xapi/Dockerfile

```diff
@@ -13,11 +18,11 @@ RUN cd /tmp && curl --compressed -L -o "xapi-service-${VERSION}.tar.gz" "https:/
 # Install xapi-service
 RUN cd /app \
     && rm package-lock.json \
-    && yarn install \
+    && yarn install --ignore-engines \
```

Comment:
The `npm_config_build_from_source` trick is not required?

Reply:
no only for learning locker.